src/glyph_position.rs

```diff
@@ -30,7 +30,7 @@ where
 }
 
 /// The position and advance of a glyph.
-#[derive(Debug, Default, Copy, Clone)]
+#[derive(Copy, Clone, Eq, Debug, Default)]
```

Comment:
Probably drop `Default` from this one too.

Reply:
`Default` was there to begin with, but alright